src/ring_theory/ideal/operations.lean

```diff
@@ -1302,7 +1302,7 @@ variables {R : Type u} [comm_semiring R]
 def is_primary (I : ideal R) : Prop :=
 I ≠ ⊤ ∧ ∀ {x y : R}, x * y ∈ I → x ∈ I ∨ y ∈ radical I
 
-theorem is_primary.to_is_prime (I : ideal R) (hi : is_prime I) : is_primary I :=
+theorem is_prime.to_is_primary {I : ideal R} (hi : is_prime I) : is_primary I :=
```

Comment:
```suggestion
theorem is_prime.is_primary {I : ideal R} (hi : is_prime I) : is_primary I :=
```
Usually we don't even include a `to_` prefix in such dot-notation. Writing `hi.is_primary` is quite pleasant, I think.
Thanks for this PR!